src/risedevtool/src/bin/risedev-dev.rs

```diff
@@ -320,7 +322,7 @@ fn task_main(
                     ExecuteContext::new(&mut logger, manager.new_progress(), status_dir.clone());
                 PostgresService::new(c.clone()).execute(&mut ctx)?;
                 let mut task =
-                    risedev::ConfigureTcpNodeTask::new(c.address.clone(), c.port, c.user_managed)?;
+                    risedev::TcpReadyCheckTask::new(c.address.clone(), c.port, c.user_managed)?;
```

Comment:
Could you please also add log ready check for pg service? @fuyufjh encountered a situation where meta panics when starting before pg is fully ready.

Reply:
Interesting. Will investigate.

Reply:
And `user_managed` is not correctly handled here. Can you help to fix it as well? 🧎🏿